subgraph.template.yaml

```diff
@@ -120,7 +147,9 @@ dataSources:
           file: ./abis/BulkIssue.json
       file: ./src/bulk-mapping.ts
       eventHandlers:
-        - event: BatchPrepared(indexed bytes32)
+        - event: BatchPrepared(indexed bytes32,indexed address)
           handler: handleBatchPrepared
-        - event: CreateSRRWithProof(indexed bytes32,bytes32)
+        - event: CreateSRRWithProof(indexed bytes32,indexed uint256,bytes32)
           handler: handleCreateSRRWithProof
+        - event: MigrateBatch(indexed bytes32,bytes32[],address,uint256,uint256)
+          handler: handleMigrateBatch
```

Comment:
@chatch I think we might need to sync the subgraph for 1.2.0 changes you made here as well. https://github.com/startbahn/startrail/pull/337/files